### PR TITLE
Featrue/zksync 2

### PR DESCRIPTION
**Pull request by ljh88hjl**
zksync contract

### CODE REVIEW

File: chain33.toml [hunk 1]
```diff
@@ -42,11 +42,11 @@ isParaChain=false
 enableTxQuickIndex=true
 enableReExecLocal=true
 # 使能精简localdb
-enableReduceLocaldb=true
+enableReduceLocaldb=false
```
Comment (mdj33) [inline chain33.toml:45]: chain33.toml  原始配置恢复最初配置

File: plugin/dapp/zksync/executor/zksyncOption.go [hunk 2]
```diff
@@ -0,0 +1,1385 @@
...
+	//TODO set chainID
+	lastPriority, err := getLastEthPriorityQueueID(a.statedb, 0)
+	if err != nil {
+		return nil, errors.Wrapf(err, "get eth last priority queue id")
+	}
+	lastPriorityId, ok := big.NewInt(0).SetString(lastPriority.GetID(), 10)
+	if !ok {
+		return nil, errors.Wrapf(types.ErrInvalidParam, fmt.Sprintf("getID =%s", lastPriority.GetID()))
+	}
+	//if lastPriorityId.Int64()+1 != payload.GetEthPriorityQueueId() {
```
Comment (mdj33) [inline plugin/dapp/zksync/executor/zksyncOption.go:86]: 去除comment

File: plugin/dapp/zksync/executor/zksyncOption.go [hunk 3]
```diff
@@ -0,0 +1,1385 @@
...
+	lastPriority, err := getLastEthPriorityQueueID(a.statedb, 0)
+	if err != nil {
+		return nil, errors.Wrapf(err, "get eth last priority queue id")
+	}
+	lastId, ok := big.NewInt(0).SetString(lastPriority.GetID(), 10)
+	if !ok {
+		return nil, errors.Wrapf(types.ErrInvalidParam, fmt.Sprintf("getID =%s", lastPriority.GetID()))
+	}
+
+	//if lastId.Int64()+1 != payload.GetEthPriorityQueueId() {
```
Comment (mdj33) [inline plugin/dapp/zksync/executor/zksyncOption.go:1107]: 去除comment

**Review by 33cn — APPROVED** [review]

**Pull request merged**